src/fileformats/iof_course_export.cpp

```diff
@@ -97,11 +98,36 @@ void IofCourseExport::writeXml(const PathObject& object)
 	}
 	{
 		XmlElementWriter event(*xml, QLatin1String("RaceCourseData"));
+		writeMap();
 		writeControls(object.getRawCoordinateVector());
 		writeCourse(object.getRawCoordinateVector());
 	}
 }
 
+void IofCourseExport::writeMap()
+{
+	XmlElementWriter map_data(*xml, QLatin1String("Map"));
+	xml->writeTextElement(QLatin1String("Scale"), QString::number(map->getScaleDenominator()));
+	
+	// IOF.xsd proposes for 'MapPositionTopLeft': "The position of the map's top left corner given in the map's coordinate system, usually (0, 0)."
+	// With this the bottom right position is determined by the (negative) size of the map extent.
+	// Alternatively use real coordinates from the map_extent structure below.
+	// NOTE: templates are not considered when determining the map extent.
```

Comment:
Hm, the current file can be just the course, with the actual map being a template.

Reply:
That's true. With the current demand for the scale information, the map positions came in as well but I doubt that anyone or any app would really use them right now.
However, I am thinking about whether to change the map positions to use the real map positions instead shifting the top left to (0,0).
Unfortunately I found no example for 'Map' settings on [Github](https://github.com/international-orienteering-federation/datastandard-v3/tree/master)

Reply:
@dg0yt: I now consider templates (if visible) and changed the map positions to use the real map positions instead.